int32 ObjectDetected
string Object
